src/app/shared/services/user-session.service.ts

```diff
@@ -61,7 +61,11 @@ export class UserSessionService implements OnDestroy {
 
   refresh(): Observable<void> {
     const refresh$ = this.currentUserService.refresh().pipe(shareReplay(1));
-    refresh$.subscribe(() => this.userProfileUpdated$.next());
+    refresh$.subscribe({
+      next: () => this.userProfileUpdated$.next(),
+      // Required for avoid to show Sentry modal
+      error: () => {},
```

Comment:
@ThomasJuster why not doing the "usual"
`if (!(err instanceof HttpErrorResponse)) throw err;` here?
Don't we have the same case as usual where if it is not a request error but a JS one, we want to report it ?

Reply:
Ok. fixed the comment to make it clear.